docs/data-types.rst

```diff
@@ -94,9 +94,20 @@ __ https://crate.io/docs/crate/reference/en/latest/general/ddl/data-types.html#c
 
 .. NOTE::
 
-    The type that ``date`` and ``datetime`` objects are mapped depends on the
+    The type that ``date`` and ``datetime`` objects are mapped to, depends on the
     CrateDB column type.
 
+.. NOTE::
+
+    Values of ``TIMESTAMP`` columns will always be stored using a ``LONG`` type,
+    representing the `Unix time`_ (epoch) timestamp, i.e. number of seconds which
+    have passed since 00:00:00 UTC on Thursday, 1 January 1970.
+
+    This means, when inserting or updating records using timezone-aware Python
+    ``datetime`` objects, timezone information will not be preserved. If you
+    need to store it, you will need to use a separate column.
```

Comment:
@mfussenegger: This paragraph intends to reflect your suggestion. Thank you again.

-- https://crate-python--557.org.readthedocs.build/en/557/data-types.html